functions/postSurveyComplete.protected.ts

```diff
@@ -71,14 +73,15 @@ const saveSurveyInInsights = async (
     postSurveyConfigJson,
     surveyTaskAttributes,
     memory,
+    pathBuilder,
   );
 
   await surveyTask.update({ attributes: JSON.stringify(finalAttributes) });
 };
 
 const saveSurveyInHRM = async (
   postSurveyConfigJson: OneToManyConfigSpec[],
-  memory: BotMemory,
+  memory: AutopilotMemory,
   surveyTask: TaskInstance,
   surveyTaskAttributes: any,
   hrmBaseUrl: string,
```

Comment:
This means we continue saving memory (in hrm and insights) in the Autopilot format, right?

Reply:
In particular this part right here means that `saveSurveyInHRM` function in `functions/postSurveyComplete.protected.ts` will use Autopilot format, but I duplicated the same functions for HRM and Insights in `/channelCapture/channelCaptureHandlers.private.ts`, since the first ones will be removed soon (once we are all in Lex).